.github/workflows/ci-cd.yaml

```
@@ -78,7 +78,48 @@ jobs:
           PLATFORM_BASE_URL: ${{ secrets.PLATFORM_BASE_URL }}
           PLATFORM_SUBSCRIPTION_KEY: ${{ secrets.PLATFORM_SUBSCRIPTION_KEY }}
           SLACK_URL: ${{ secrets.SLACK_URL }}
+          SBL_BRIDGE_BASE_URL: ${{ secrets.SBL_BRIDGE_BASE_URL }}
+
+  deploy-at22:
+    name: deploy at22
+    runs-on: ubuntu-latest
+    environment: test
+    if: always() && !failure() && !cancelled() 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)